### ISSUE TITLE
Can't search by namespace prefixes (error 424)

### ISSUE THREAD

**Issue by Ivan-Shestakov** [thomsonreuters/CM-Well#490]
Whenever searching for namespace prefix existence we're getting 
**HTTP Status 424 Failed Dependency**
curl -v 'http://cm-well-host/meta/ns/?op=search&qp=prefix:rdf&format=json&pretty'
``` JSON
{"success":false,"message":"failed to evaluate given qp [RawSingleFieldFilter(Must,Contains,Right(NnFieldKey(prefix)),Some(rdf))]"}
```
This reproduces on both newly installed and upgraded environments.

**Comment by hochgi** [comment]
same as #474

**Comment by Ivan-Shestakov** [comment]
Validated the fix on environment with git commit hash ea325a6c1ef1ca927dfbdf27706d58894baaa447 installed
Can search successfully with `&qp=prefix:rdf` and `&qp=url:RDF` finding appropriate results.

Moving to Done.
